todo/src/component/TodoList.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import styled from 'styled-components';
+import { addTODO, toggleTodo, deleteTodo} from '../actions/actionCreators';
+import Todo from './Todo';
+import Form from './Form';
+
+const Container = styled.div`
+margin: 0 auto;
+`;
+
+const Header = styled.div`
+@import url("https://fonts.googleapis.com/css?family=Lobster&display=swap");
+  font-family: "Lobster", cursive;
+  font-size: 2rem;
+  font-weight: lighter;
+  padding: 0 0 0 1rem;
+  margin: 0 0 3rem 0;
+`;
+
+class TodoList extends React.Component{
+
+    state = {
+        newTodo: ''
+    };
+
+    addTODO = e => {
+        e.preventDefault();
+        this.props.addTODO(this.state.newTodo);
+        this.setState({ newTodo: '' })
+    }
+
+    completed = (id) =>{
+        this.props.toggleTodo(id)
```

Comment:
the functions from the props could probably just be called inline as this is just a function calling a function